### DISCUSSION TITLE
Battleye Bypass

### DISCUSSION THREAD

**Discussion by GitCat3** [Mr-X-GTA/YimMenu#12] [General]
Does this mod menu bypass the GTA Battleye anticheat? If so, is there a certain way I have to load it to make it do so?

**Comment by Tessio** [comment]
I will explain a little how it works:

The mod works without BattlEye, it doesn't omit the anticheat itself, but it allows to play without problems in online.

How to use it:

- Open Rockstar Launcher

- Go to settings and disable BattlEye

- Open GTA V

- Inject the Mod Menu as soon as the game is open (before entering the story mode).

- Enjoy (=

**Comment by GitCat3** [comment]
Thanks for the explanation! If you don't mind, I'm a somewhat technical person, does the mod achieve this by mimiking the network packets that Battleye sends to seem legit?

**Reply by Tessio** [reply]
In short, a hooking was used for a specific native.

`add_native_detour(NativeIndex::_GET_BATTLEYE_INIT_STATE, all_scripts::RETURN_FALSE); `

But you can always check yourself the code of this specific update in the commit:
[#3697](https://github.com/Mr-X-GTA/YimMenu/commit/220322b910e6474eefd24f22aa98f4a1ade57630#diff-29e4110658e3ddaea006c4f4f796b39aa53458c59c4447a82c3002d284eec7a0R101)

**Reply by GitCat3** [reply]
Thanks for the explanation!

**Comment by GitCat3** [comment]
Also what do you recommend for injection tool?

**Reply by L7NEG** [reply]
Any will work fate injector on github or xenos injector on github too or even the stand launchpad
